### PR TITLE
[16.0][MIG] account_banking_mandate_contact: Migration to 16.0.

### PR DESCRIPTION
**Pull request by Tisho99**

### CODE REVIEW

**Comment by Tisho99** [comment]
@ValentinVinagre @HaraldPanten

File: account_banking_mandate_contact/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    contact_mandate_id = fields.Many2one(
+        comodel_name="account.banking.mandate",
+        string="Contact Mandate",
+    )
+
+    def _compute_valid_mandate_id(self):
+        procesed_partners = self.browse()
```
Comment (ValentinVinagre) [inline account_banking_mandate_contact/models/res_partner.py]:
```
def _compute_valid_mandate_id(self):
        procesed_partners = self.browse()
        for partner in self.filtered(lambda x: x.contact_mandate_id.state == "valid"):
                partner.valid_mandate_id = partner.contact_mandate_id
                procesed_partners |= partner
        return super(ResPartner, self - procesed_partners)._compute_valid_mandate_id()
```

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

File: account_banking_mandate_contact/models/res_partner.py (hunk 1 above)
Comment (Tisho99) [reply account_banking_mandate_contact/models/res_partner.py]: I have uploaded your suggestions, but I have also removed the procesed_partners field to partner_to_process

**Comment by ValentinVinagre** [comment]
@HaraldPanten could u review please? 😄

**Review by HaraldPanten — APPROVED** [review]
Functional review in runboat. LGTM

**Pull request merged**